requirements_all.txt

```diff
@@ -2776,11 +2776,13 @@ temperusb==1.6.1
 # homeassistant.components.tensorflow
 # tensorflow==2.5.0
 
-# homeassistant.components.tesla_fleet
 # homeassistant.components.teslemetry
 # homeassistant.components.tessie
 tesla-fleet-api==0.7.3
 
+# homeassistant.components.tesla_fleet
+tesla-fleet-api==0.7.4
```

Comment:
You can't bump just one integration, you have to bump Tessie and Teslemetry too.

Reply:
I just pushed a commit that fixes this.